src/Bluejay.asm

```diff
@@ -1092,16 +1111,19 @@ exit_run_mode_stall_done_beep:
     ljmp arming_begin                   ; Go back and wait for arming
 
 exit_run_mode_no_stall:
+    ; Enable all interrupts (disabled above, in exit_run_mode)
+    setb IE_EA
+
     ; Clear extended DSHOT telemetry flag if turtle mode is not active
     ; This flag is also used for EDT safety arm flag
     ; We don't want to deactivate extended telemetry during turtle mode
     ; Extended telemetry flag is important because it is involved in
     ; EDT safety feature. We don't want to disable EDT arming during
     ; turtle mode.
-    jb Flag_User_Reverse_Requested, exit_run_mode_no_stall_beep
+    jb Flag_User_Reverse_Requested, exit_run_mode_no_stall_no_beep
     clr Flag_Ext_Tele
 
-exit_run_mode_no_stall_beep:
+exit_run_mode_no_stall_no_beep:
     ; Clear stall counter
     mov  Startup_Stall_Cnt, #0
```

Comment:
Related to above, why `mov` instead of `clr`?

Reply:
clr can only be used with A